### PR TITLE
chore(common): clean up unused code

### PR DESCRIPTION
**Pull request by eunjae-lee**
## What?

This PR cleans up unused code.

## Why?

EXT-1732
EXT-1723
EXT-1721
EXT-1724

### CODE REVIEW

File: packages/cli/templates/vue2/index.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
+<!-- DO NOT EDIT THIS FILE. -->
+<!-- This file mimics the production environment, -->
+<!-- but editing it won't have any effect -->
+<!-- on the version deployed in production. -->
```
Comment (eunjae-lee) [inline packages/cli/templates/vue2/index.html:4]: Once we finalize this message, I'll replicate it to other templates. Feel free to suggest edits.

Comment (BibiSebi) [reply packages/cli/templates/vue2/index.html:4]: maybe we should state why it should not be edited and what effects the change might have ?

Comment (BibiSebi) [reply packages/cli/templates/vue2/index.html:4]: Something like: 

This file mimics the production environment. A change to this file could result in issues when testing your field plugin in dev-mode.

Comment (eunjae-lee) [reply packages/cli/templates/vue2/index.html]: https://github.com/storyblok/field-plugin/pull/222/commits/603bd8882ae346f8f3ebc25d973975a808f29d09

nice, I made a bit of change

**Review by BibiSebi — APPROVED** [review]
Looking good, lets gooooo 💯

**Pull request merged**